docs/plans/KILO_GASTOWN_SPEC.md

```
@@ -0,0 +1,226 @@
+#### Bead Types
+
+| Type | Purpose | Example |
+|------|---------|---------|
+| `issue` | Task or bug | "Implement feature X" |
+| `escalation` | Blocked or problematic work | "Branch does not exist" |
+| `merge_request` | Code review request | PR review from agent |
+| `triage` | Routing decision needed | "GUPP force stop" |
+
+#### Bead Lifecycle
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)